demisto_sdk/commands/init/initiator.py

```diff
@@ -283,6 +324,29 @@ def get_selected_template(self, template: str = "") -> str:
         # if reached here it is a pack init - will be used again if user decides to create an integration
         return template
 
+    def process_files(self, file_list: Set[str], path_of_template) -> None:
+        """Creates empty files according to a given list or from a template if exist.
+
+        Args:
+            file_list (List(str)): A list with path and names of the files.
+        """
+        for file in file_list:
+            file_path = os.path.join(self.full_output_path, file)
+            template_path = os.path.join(path_of_template, file)
+            dir_path = os.path.dirname(file_path)
```

Comment:
Do not use `os.path` in new code (or at all), use `pathlib.Path` instead. See the recorded video session.